### ISSUE TITLE
Specify parameter by its tree location rather than custom name

### ISSUE THREAD

**Issue by MaxGhenis** [PolicyEngine/policyengine#411]
I'm guessing there's a strong reason to add the custom parameter names rather than the tree location, e.g. making the URLs work, but just dropping this issue here in case it's worth exploring. All else equal it'd be nice to be able to skip the parallel identifier.

**Comment by nikhilwoodruff** [comment]
Yeah, though my initial thought was that it'd make the URLs significantly longer

**Comment by MaxGhenis** [comment]
True, though it'd be nice to have an URL shortener at some point.

**Comment by nikhilwoodruff** [comment]
We now can do this (e.g. see the pensioner tax reform exemption, which doesn't have a `name` attribute). It's not completely unique (though in practice errors aren't likely)
